InSAR/multiburst/multiburst_insar.ipynb

```diff
@@ -0,0 +1,261 @@
+    "3. A Multiburst InSAR command line tool, which this notebook discusses in greater detail.\n",
+    "\n",
+    "Later this year, we plan to extend our burst InSAR HyP3 job to be able to create InSAR products from multiple bursts. This will be a huge improvement in the utility of the HyP3 job, because most deformation patterns span somewhere between 1 and 20 bursts.\n",
+    "\n",
+    "As a first step towards offering this capability via HyP3, we've added the ability to perform multiburst InSAR to our [`HyP3-ISCE2`](https://github.com/asfhyp3/hyp3-isce2) package, which you can run for yourself today. This notebook demos the current functionality.\n",
+    "\n",
+    "For this notebook we will be comparing multiburst and standard SLC InSAR for the [2019 Ridgecrest Earthquake](https://earthquake.usgs.gov/storymap/index-ridgecrest.html) using the ascending pass of Sentinel-1."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9a097ea0-4e88-4525-b9ba-498366896979",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "First, let's set up some directories for running the analyses"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "8552abeb-13e3-4d0e-856a-c8ff2dfe64ff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "\n",
+    "base = Path.cwd()\n",
+    "slc_upper = base / 'slc_upper'\n",
+    "slc_lower = base / 'slc_lower'\n",
+    "burst = base / 'burst'\n",
+    "\n",
+    "for directory in [slc_upper, slc_lower, burst]:\n",
+    "    directory.mkdir(parents=True, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64e23916-3bd1-4a3c-a8ef-ad99450eda90",
+   "metadata": {},
+   "source": [
+    "## SLC\n",
+    "\n",
+    "Let's take a look at how to create ascending pass Sentinel-1 InSAR data for the Ridgecrest Earthquake.\n",
+    "\n",
+    "The Ridgecrest Earthquake deformation is located at the boundary between two Sentinel-1 SLC frames, so we'll have to produce two InSAR pairs. This leads to longer runtimes and to producing InSAR data for areas we're not interested in. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f028c94-6ade-49b8-807d-3d75728fcaf1",
+   "metadata": {},
+   "source": [
+    "## Upper SLC Processing\n",
+    "\n",
+    "[View the input Sentinel-1 scenes in Vertex](https://search.asf.alaska.edu/#/?maxResults=250&zoom=7.504&center=-114.798,33.203&polygon=POLYGON((-117.8581%2035.2613,-117.0303%2035.2613,-117.0303%2035.9371,-117.8581%2035.9371,-117.8581%2035.2613))&searchType=Geographic%20Search&resultsLoaded=false&granule=S1A_IW_SLC__1SDV_20190704T015049_20190704T015116_027961_03283A_191E-SLC&productTypes=SLC&flightDirs=Ascending&start=2019-07-03T05:00:00Z&end=2019-07-25T04:59:59Z&path=64-64&frame=114-114)\n",
```

Comment:
Link doesn't render in MD because of parenthesis in link

Reply:
It's not nearly as pretty, but this will render correctly:
```
 <a href="https://search.asf.alaska.edu/#/?maxResults=250&zoom=7.504&center=-114.798,33.203&polygon=POLYGON((-117.8581%2035.2613,-117.0303%2035.2613,-117.0303%2035.9371,-117.8581%2035.9371,-117.8581%2035.2613))&searchType=Geographic%20Search&resultsLoaded=false&granule=S1A_IW_SLC__1SDV_20190704T015049_20190704T015116_027961_03283A_191E-SLC&productTypes=SLC&flightDirs=Ascending&start=2019-07-03T05:00:00Z&end=2019-07-25T04:59:59Z&path=64-64&frame=114-114" > View the input Sentinel-1 scenes in Vertex  </a>
```